src/vm/method.hpp

```diff
@@ -258,6 +258,17 @@ class MethodDesc
         return (m_bFlags2 & enum_flag2_HasStableEntryPoint) != 0;
     }
 
+    inline BOOL HasStableEntryPoint1()
+    {
+        LIMITED_METHOD_DAC_CONTRACT;
+
+#if defined(FEATURE_JIT_PITCHING)
+        if (IsPitchable() && HasTemporaryEntryPoint())
+           return FALSE;
+#endif
+        return HasStableEntryPoint();
+    }
+
```

Comment:
No worries, this is your feature so you get to set the pace, whether that is having it ready tomorrow or next year ; ) Enjoy your vacation!